Rock.Encryption/Symmetric/RNG.cs

```diff
@@ -1,12 +1,16 @@
 using System.Security.Cryptography;
 using System.Threading;
 
+#if ROCKLIB
+namespace RockLib.Encryption.Symmetric
+#else
 namespace Rock.Encryption.Symmetric
+# endif
 {
     internal static class RNG
     {
-        private static readonly ThreadLocal<RNGCryptoServiceProvider> _instance =
-            new ThreadLocal<RNGCryptoServiceProvider>(() => new RNGCryptoServiceProvider());
+        private static readonly ThreadLocal<RandomNumberGenerator> _instance =
```

Comment:
RNGCrypto wasn't available in standard, but result was the same with straight RandomNumberGenerator.